apps/api-gateway/src/main/resources/application-docker.yml

```
@@ -0,0 +1,19 @@
+quarkus:
+  log:
+    level: INFO
+  http:
+    port: 8080
+  rest-client:
+    http2: true
+  mongodb:
+    connection-string: mongodb://localhost:27017
+    database: gateway
+
+gateway:
+  routes:
+    FINANCES:
+      hostname: finances
+      port: 8080
+    INVENTORY:
+      hostname: localhost
```

Change to correct hostname 
```suggestion
      hostname: inventory
```